File: OOPS_LAB/Program_10/book.java
class BookDetails{
    private String title;
    private String author;
    private double price;

    public BookDetails(String title, String author, double price) {
        this.title=title;
        this.author=author;
        this.price=price;
    }

    public void display()
    {
        System.out.println("\tBOOK DETAILS");
        System.out.println("Titel : "+this.title);
        System.out.println("Author : "+this.author);
        System.out.println("Price : "+this.price);
    }
}

public class book {
    public static void main(String[] args) {
        BookDetails book1 = new BookDetails("Kya Bole", "Ashfak Alam", 1000);
        BookDetails book2 = new BookDetails("DSA", "Ashfak Alam", 999);
        BookDetails book3 = new BookDetails("COA", "Ashfak Alam", 1999);
        book1.display();
        book2.display();
        book3.display();
    }
}

File: OOPS_LAB/Program_10/book.cpp
#include<iostream>
#include<print>
using namespace std;

class Book{
    string title;
    string author;
    float price;
  public:
    Book(string tit, string auth, float pri) : title(tit),author(auth),price(pri){}

    void display()
    {
        println("Title : {}",title);
        println("Author : {}",author);
        println("Price : RS.{}",price);
    }
};

int main(int argc, char const *argv[])
{
    Book book1("DSA","Ashfak Alam",1999);
    book1.display();
    return 0;
}